index.html

```diff
@@ -11,7 +11,7 @@
     <h1>Lab Week 1 Demo!</h1>
     <p>This is a test!</p>
     <p>Test branch</p>
-    <p>Change to the branch</p>
+    <p>Sabrina</p>
```

Comment:
This change is correct because we were instructed to add our names to the HMTL file in the instructions for Lab 1